paparazzi/paparazzi-gradle-plugin/src/test/projects/layout-direction-xml/src/test/java/app/cash/paparazzi/plugin/test/LayoutDirectionXmlTest.kt

```diff
@@ -0,0 +1,43 @@
+package app.cash.paparazzi.plugin.test
+
+import app.cash.paparazzi.DeviceConfig
+import app.cash.paparazzi.Paparazzi
+import com.android.resources.LayoutDirection
+import com.google.testing.junit.testparameterinjector.TestParameter
+import com.google.testing.junit.testparameterinjector.TestParameterInjector
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(TestParameterInjector::class)
+class LayoutDirectionXmlTest(
+  @TestParameter localeAndDirection: LocaleAndDirection
+) {
+
+  enum class LocaleAndDirection(
+    val tag: String,
+    val direction: LayoutDirection
+  ) {
+    DefaultRtl(
+      tag = "",
+      direction = LayoutDirection.RTL
+    ),
+    AR(
+      tag = "ar",
+      direction = LayoutDirection.LTR
+    );
+  }
```

Comment:
```suggestion
  enum class LocaleAndDirection(
    val tag: String?,
    val direction: LayoutDirection
  ) {
    DefaultRtl(
      tag = null,
      direction = LayoutDirection.RTL
    ),
    AR(
      tag = "ar",
      direction = LayoutDirection.LTR
    );
  }
```

Reply:
nit: drop `-xml` from name of test projects

Reply:
food for thought: should the test projects `layout-direction` and `locale-qualifier` be merged?

Reply:
Dropped prefixes here: 1de4494f47886bff78fd3f3a92714a8a6e080c59
This can be one test with 4 snapshots to verify, but this drops granularity a bit. But I don't have any strong opinion here, so up to you.